core/Updater.php

```diff
@@ -241,7 +241,7 @@ public function getSqlQueriesToExecute()
 
                 $className = $this->getUpdateClassName($componentName, $fileVersion);
                 if (!class_exists($className, false)) {
-                    throw new \Exception("The class $className was not found in $file");
+                    throw new \Error("The class $className was not found in $file");
```

Comment:
Could you add a comment here about the use of `\Error`? It seems like an easy thing to forget.

Reply:
sure

Reply:
done